packages/react/src/__tests__/ReactElement-test.internal.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+let React;
+let ReactDOM;
+let ReactFeatureFlags;
+
+describe('ReactElement', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactFeatureFlags.warnAboutCreateElement = true;
+    React = require('react');
+    ReactDOM = require('react-dom');
+  });
+
+  it('should warn when createElement is accessed ', () => {
+    const container = document.createElement('div');
+    expect(() =>
+      ReactDOM.render(React.createElement('div'), container),
+    ).toWarnDev(
+      'Warning: div: You are most likely getting this warning because ' +
```

Comment:
Suggestion: Add an extra sentence to the beginning like

> A `key` attribute on a JSX element should come before an object spread

The rest of the message is good.